.gitignore

```diff
@@ -1,3 +1,84 @@
+<<<<<<< HEAD
```

Comment:
- [ ] conflicts should be resolved before pushing

Reply:
Will be fixed

Reply:
@smokhov just to clarify, you want us to now update the pull request with PM2.3 content instead of just PM1 content?

Thank you for your time.